=================================================================================

This patch solves a problem that arises when compiling support for VDE
transport.

TECHNICAL INFORMATION:

Applies to vanilla kernel 2.6.26.5.

=================================================================================

diff -Naur linux-2.6.26.3/arch/um/drivers/vde_user.c linux-2.6.26.3-changed/arch/um/drivers/vde_user.c
--- linux-2.6.26.3/arch/um/drivers/vde_user.c	2008-08-20 20:11:37.000000000 +0200
+++ linux-2.6.26.3-changed/arch/um/drivers/vde_user.c	2008-10-13 12:42:54.000000000 +0200
@@ -78,7 +78,7 @@
 {
 	struct vde_open_args *args;
 
-	vpri->args = kmalloc(sizeof(struct vde_open_args), UM_GFP_KERNEL);
+	vpri->args = uml_kmalloc(sizeof(struct vde_open_args), UM_GFP_KERNEL);
 	if (vpri->args == NULL) {
 		printk(UM_KERN_ERR "vde_init_libstuff - vde_open_args "
 		       "allocation failed");
